src/components/pages/Offer/StocksManager/StockItem/decorators/__specs__/adaptBookingLimitDatetimeGivenBeginningDatetime.spec.js

```diff
@@ -71,13 +70,32 @@ describe('src | components | pages | Offer | StockManager | StockItem | decorato
         const result = updateBookingLimitDatetime({
           beginningDatetime,
           bookingLimitDatetime,
-          isEvent
+          isEvent,
         })
 
         // then
         expect(result.bookingLimitDatetime).toStrictEqual('2019-04-20T19:00:00.000Z')
       })
     })
+
+    describe('when bookingLimiteDateTime is empty', () => {
+      it('should leave bookingLimitDatetime empty', () => {
+        // given
+        const isEvent = true
+        const beginningDatetime = '2019-04-20T19:00:00.000Z'
+        const bookingLimitDatetime = ''
+
+        // when
+        const result = updateBookingLimitDatetime({
+          beginningDatetime,
+          bookingLimitDatetime,
+          isEvent,
+        })
+
+        // then
+        expect(result.bookingLimitDatetime).toStrictEqual('')
```

Comment:
```suggestion
        expect(result.bookingLimitDatetime).toBe('')
```
Si ce n'est pas un `object` ou un `array`, `toBe` est plus approprié. Ce qui serait bien, c'est un petit BSR.